app/src/components/buttons/toggleButton/toggleButton.jsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2017 EPAM Systems
+ *
+ *
+ * This file is part of EPAM Report Portal.
+ * https://github.com/reportportal/service-ui
+ *
+ * Report Portal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Report Portal is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Report Portal.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './toggleButton.scss';
+
+const cx = classNames.bind(styles);
+
+export class ToggleButton extends Component {
+  static propTypes = {
+    items: PropTypes.object,
```

Comment:
```
items: PropTypes.arrayOf(PropTypes.shapeOf({
  value: PropTypes.string.isRequired,
  label: PropTypes.string.isRequired,
})),
```

Also, do we need to disable buttons in some cases? If yes, add `disabled` attribute to the item shape.